cob_openni2_tracker/package.xml

```diff
@@ -1,70 +1,50 @@
 <?xml version="1.0"?>
-<package>
-  <name>cob_openni2_tracker</name>
-  <version>0.1.0</version>
-  <description>
-    The openni_tracker broadcasts the OpenNI skeleton frames using tf.
-    This new version of the openni_tracker uses OpenNI2 and Nite2</description>
-  <maintainer email="richard.bormann@ipa.fraunhofer.de">Richard Bormann</maintainer>
-  
-  <license>BSD</license>
-  <url type="website">http://www.ros.org/wiki/openni_tracker</url>  
-  <author email="marcus.liebhardt@yujinrobot.com">Marcus Liebhardt</author> 
-  <author>Olha Meyer</author>
+<package format="2">
+	<name>cob_openni2_tracker</name>
+	<version>0.1.0</version>
+	<description>
+		The openni_tracker broadcasts the OpenNI skeleton frames using tf.
+		This new version of the openni_tracker uses OpenNI2 and Nite2</description>
+	<maintainer email="richard.bormann@ipa.fraunhofer.de">Richard Bormann</maintainer>
+	
+	<license>BSD</license>
+	<url type="website">http://www.ros.org/wiki/openni_tracker</url>	
+	<author email="marcus.liebhardt@yujinrobot.com">Marcus Liebhardt</author> 
+	<author>Olha Meyer</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
+	<buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>cmake_modules</build_depend>
-  <build_depend>libusb-1.0-dev</build_depend>
-  <!--  <build_depend>libopenni-nite-dev</build_depend> -->
-  <build_depend>libopenni-sensor-primesense-dev</build_depend> -->
-  <build_depend>geometry_msgs</build_depend>
-  <build_depend>sensor_msgs</build_depend>
-  <build_depend>orocos_kdl</build_depend>
-  <build_depend>roscpp</build_depend>
-  <build_depend>roslib</build_depend>
-  <build_depend>visualization_msgs</build_depend>
-  <build_depend>tf</build_depend>
-  <build_depend>cob_perception_msgs</build_depend> 
-  <build_depend>libopenni2-dev</build_depend>
-  <build_depend>libnite2</build_depend>
-  <build_depend>nodelet</build_depend>
-  <build_depend>pcl_ros</build_depend>
-  <build_depend>image_transport</build_depend>
-  <build_depend>cv_bridge</build_depend>
-  <build_depend>std_msgs</build_depend>
-  
-  <build_depend>libopencv-dev</build_depend>
-  <build_depend>glut</build_depend>
-  <build_depend>opengl</build_depend>
+	<depend>cmake_modules</depend>
+	<depend>libusb-1.0-dev</depend>
+	<!--	<depend>libopenni-nite-dev</depend> -->
+	<!--	<depend>libopenni-sensor-primesense-dev</depend> -->
```

Comment:
This is what travis currently complains about.

do you need these for compiling this node? If yes, then it should be a `depend`
(the rosdep is listed [here](https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L1763))